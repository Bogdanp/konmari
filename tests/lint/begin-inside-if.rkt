#lang racket/base

(if #t
    #f
    (let ()
      1))

(if #t
    (begin #f)
    #t)
