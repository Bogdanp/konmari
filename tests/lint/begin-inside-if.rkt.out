/begin-inside-if.rkt:3:1:warning:use a cond expression instead of nesting begin or let inside an if
/begin-inside-if.rkt:8:1:warning:use a cond expression instead of nesting begin or let inside an if
